modules/client-configuration/pages/autoinst-pxeboot.adoc

```diff
@@ -0,0 +1,99 @@
+[[installation-via-pxe-boot]]
+= Installation via PXE Boot
+
+Here is how a network installation goes:
```

Comment:
```suggestion
During a PXE boot network installation, these steps occur:
```

Probably better wording here but I'm not sure what.

Reply:
i'll try to just remove the part after the comma